org.shadered.SHADERed.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: org.shadered.SHADERed
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: shadered
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --filesystem=home
+  - --device=all
+
+modules:
+  - name: assimp
+    buildsystem: cmake-ninja
+    cleanup:
+      - "*.hpp"
+      - "*.h"
+      - "*.inl"
```

Comment:
You can add a top-level `cleanup`, so you won't need to repeat it for every module. Also you probably should cleanup static libraries.